src/battle_message.c

```diff
@@ -2092,7 +2092,19 @@ void BufferStringBattle(enum StringID stringID, u32 battler)
                 else if (gBattleTypeFlags & BATTLE_TYPE_MULTI)
                 {
                     if (gBattleTypeFlags & BATTLE_TYPE_RECORDED)
-                        stringPtr = sText_TwoLinkTrainersWantToBattlePause;
+                    {
+                        if(TESTING && gBattleTypeFlags & BATTLE_TYPE_MULTI)
+                        {
+                            if(!(gBattleTypeFlags & BATTLE_TYPE_TWO_OPPONENTS))
+                                stringPtr = sText_Trainer1WantsToBattle;
+                            else
+                                stringPtr = sText_TwoTrainersWantToBattle;
+                        }
+                        else if(!(gBattleTypeFlags & BATTLE_TYPE_TWO_OPPONENTS))
+                            stringPtr = sText_LinkTrainerWantsToBattlePause;
+                        else
+                            stringPtr = sText_TwoLinkTrainersWantToBattlePause;
```

Comment:
```suggestion
                        {
                            stringPtr = sText_LinkTrainerWantsToBattlePause;
                        }
                        else
                        {
                            stringPtr = sText_TwoLinkTrainersWantToBattlePause;
                        }
```
All parts of an if-else chain should have brackets if one of the parts have